quickwit/quickwit-cli/src/index.rs

```diff
@@ -192,6 +192,8 @@ pub fn build_index_command<'a>() -> Command<'a> {
                         .display_order(1),
                     arg!(--"dry-run" "Executes the command in dry run mode and only displays the list of splits candidates for deletion.")
                         .required(false),
+                    arg!(--"safe-mode" "Prompt for confirmation before deleting an index.")
```

Comment:
I think `--safe` might be a better name for this flag, IMO it gets the point across more directly.

Reply:
Not a problem, I could change the name no problem. Aaaaaaaaand... done!